example/package.json

```diff
@@ -10,7 +10,7 @@
   "dependencies": {
     "core-js": "^3.2.1",
     "evergreen-ui": "^4.18.1",
-    "evermut": "^1.2.7-beta7",
+    "evermut": "^1.3.1-beta1",
```

Comment:
is this the latest version ?

Reply:
for now yes - 1.4.0-beta1 will be the next published version